src/GovUk.Education.ExploreEducationStatistics.Admin/Migrations/UsersAndRolesMigrations/20220527101017_EES3388_RenameTestUserEmails_upsert.sql

```diff
@@ -0,0 +1,34 @@
+UPDATE AspNetUsers SET UserName = 'ees-test.bau1@education.gov.uk' WHERE UserName = 'ees-bau1@education.gov.uk';
+UPDATE Users SET Email = 'ees-test.bau1@education.gov.uk' WHERE Email = 'ees-bau1@education.gov.uk';
+
+UPDATE UserInvites SET Email = 'ees-test.bau1@education.gov.uk' WHERE Email = 'bau1@azurehiveitco.onmicrosoft.com';
+
+
+UPDATE AspNetUsers SET Email = 'EES-test.BAU2@education.gov.uk' WHERE Email = 'ees-bau2@education.gov.uk';
+UPDATE AspNetUsers SET NormalizedEmail = 'EES-TEST.BAU2@education.gov.uk' WHERE NormalizedEmail = 'EES-BAU2@education.gov.uk';
+UPDATE AspNetUsers SET NormalizedUserName = 'EES-TEST.BAU2@education.gov.uk' WHERE NormalizedUserName = 'EES-BAU2@EDUCATION.GOV.UK';
+UPDATE AspNetUsers SET UserName = 'ees-test.bau2@education.gov.uk' WHERE UserName = 'ees-bau2@education.gov.uk';
+UPDATE Users SET Email = 'ees-test.bau2@education.gov.uk' WHERE Email = 'ees-bau2@education.gov.uk';
+UPDATE UserInvites SET Email = 'ees-test.bau2@education.gov.uk' WHERE Email = 'bau2@azurehiveitco.onmicrosoft.com';
+
+
+UPDATE AspNetUsers SET NormalizedUserName = 'EES-TEST.ANALYST1@education.gov.uk' WHERE NormalizedUserName = 'EES-ANALYST@EDUCATION.GOV.UK';
+UPDATE AspNetUsers SET Email = 'EES-test.ANALYST1@education.gov.uk' WHERE Email = 'ees-analyst1@education.gov.uk';
+UPDATE AspNetUsers SET NormalizedEmail = 'EES-TEST.ANALYST1@education.gov.uk' WHERE NormalizedEmail = 'EES-ANALYST1@education.gov.uk';
+UPDATE AspNetUsers SET UserName = 'ees-test.analyst1@education.gov.uk' WHERE UserName = 'ees-analyst1@education.gov.uk';
+UPDATE Users SET Email = 'ees-test.analyst1@education.gov.uk' WHERE Email = 'ees-analyst1@education.gov.uk';
+
+UPDATE AspNetUsers SET NormalizedUserName = 'EES-TEST.ANALYST2@education.gov.uk' WHERE NormalizedUserName = 'EES-ANALYST2@EDUCATION.GOV.UK';
+UPDATE AspNetUsers SET Email = 'EES-test.ANALYST2@education.gov.uk' WHERE Email = 'ees-analyst2@education.gov.uk';
+UPDATE AspNetUsers SET NormalizedEmail = 'EES-TEST.ANALYST2@education.gov.uk' WHERE NormalizedEmail = 'EES-ANALYST2@EDUCATION.GOV.UK';
+UPDATE AspNetUsers SET UserName = 'ees-test.analyst2@education.gov.uk' WHERE UserName = 'ees-analyst2@education.gov.uk';
+UPDATE Users SET Email = 'ees-test.analyst2@education.gov.uk' WHERE Email = 'ees-analyst2@education.gov.uk';
+
+UPDATE AspNetUsers SET NormalizedUserName = 'EES-TEST.ANALYST3@education.gov.uk' WHERE NormalizedUserName = 'EES-ANALYST3@EDUCATION.GOV.UK';
+UPDATE AspNetUsers SET Email = 'EES-test.ANALYST3@education.gov.uk' WHERE Email = 'ees-analyst3@education.gov.uk';
+UPDATE AspNetUsers SET NormalizedEmail = 'EES-TEST.ANALYST3@education.gov.uk' WHERE NormalizedEmail = 'EES-ANALYST3@EDUCATION.GOV.UK';
+UPDATE AspNetUsers SET UserName = 'ees-test.analyst3@education.gov.uk' WHERE UserName = 'ees-analyst3@education.gov.uk';
+UPDATE Users SET Email = 'ees-test.analyst3@education.gov.uk' WHERE Email = 'ees-analyst3@education.gov.uk';
```

Comment:
What's going on with the Prerelease users for this change? Are we not touching them for now because we didn't get the accounts yet?